packages/examples/package.json

```diff
@@ -0,0 +1,16 @@
+{
+  "name": "nibijs-examples",
+  "module": "examples.test.ts",
+  "type": "module",
+  "version": "0.0.1",
+  "dependencies": {
+    "@nibiruchain/nibijs": "0.21.42"
+  },
+  "engines": {
+    "node": ">=18"
+  },
+  "devDependencies": {
+    "es-main": "^1.3.0",
+    "tsx": "^4.6.2"
+  }
```

Comment:
Where and how are these devDependencies used?

Reply:
If you want logic to run when a file is called as a script, this is typically done by checking if `require.main === module'. However, that statement doesn't work consistently with different node setups.

`tsx` (poorly named) is a TypeScript executor similar to `ts-node`. It's used to run the examples as a script. It's the following line in the test workflow:
 `yarn run tsx packages/examples/examples.test.ts`

Funny enough, `tsx` is one environment where `require.main` doesn't exist. The recommended solution I found online that will work properly more consistently is to use `es-main` with the `import` keyword instead of `require`, as it will always be avaialable.

Reply:
Interesting,

I would argue, that the examples.test.ts file is already a test file, why not use jest as an executor?

As a side note a tsconfig.json could also be useful here (in relation to import's)

I'm okay with what you have setup, but I think the above solution would be cleaner